docs/site/todo-list-tutorial-controller.md

```diff
@@ -221,6 +221,102 @@ export class TodoListController {
 }
 ```
 
+And the completed TodoListTodo controller should look like follows:
```

Comment:
Is it a good idea to keep a copy of the generated code in our docs? It means that whenever we change the CLI templates, we also need to update the copies in our docs. In my experience so far, we usually forget to do that and as a result, our documentation often contains outdated code.

Can we point users to the source code file on GitHub instead? For example:

```markdown
Check out our todo-list example to see the full source code generated for TodoListTodo controller:
[src/controllers/todo-list-todo.controller.ts](https://github.com/strongloop/loopback-next/blob/master/examples/todo-list/src/controllers/todo-list-todo.controller.ts)
```